subprojects/spdk-win.wrap

```diff
@@ -0,0 +1,13 @@
+# SPDX-FileCopyrightText: Samsung Electronics Co., Ltd
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+[wrap-git]
+url = https://github.com/spdk/spdk.git
+revision = v22.05
+patch_directory = spdk-win
+clone-recursive = true
```

Comment:
I believe we can avoid the issue if we use the following instead of `[wrap-git]`:
```
[wrap-file]
source_url = https://github.com/spdk/spdk/archive/refs/tags/v22.05.tar.gz
source_filename = spdk-22.05.tar.gz
```

Reply:
Looks like there is an issue with the tar-all as it does not include the git-submodules e.g. DPDK etc.